ui/src/dashboards/components/CellEditorOverlay.tsx

```diff
@@ -20,6 +20,7 @@ import {
   addQueryAsync,
   deleteQueryAsync,
   updateEditorTimeRange as updateEditorTimeRangeAction,
+  updateScript as updateScriptAction,
```

Comment:
this renaming is a little misleading since updateScript is a function that returns an action and not an action itself

Reply:
I think I'm confused on this-- if it returns an action doesn't that make it an action?